### PR TITLE
[DOC] Add each addon documentation in it folder

### PR DESCRIPTION
**Pull request by abdounasser202**
When we have lot of modules, it's better to document each one on his folder and sumarize them in the repo readme

### CODE REVIEW

**Comment by abdounasser202** [comment]
@Yenthe666  @sswapnesh @wardm95 in this pull request, I suggest reordering the documentation

**Comment by Yenthe666** [comment]
@abdounasser202 thanks a lot for your PR, I really appreciate it 😄 I'm a bit torn about this one.
People onlytend to (or I think so) look at the main page of the repository to find into/context about the apps.
What do you think? Maybe an idea is to list the apps with a quick link on the main `README` to have a quick jump to the info page per app that you made? 🤔 
@sswapnesh what is your take?

**Comment by abdounasser202** [comment]
Yep @Yenthe666 this is what is done
In the main page of the repo there are links to the modules that shows a summary for each addon and when you click there you go to the full module explanation

**Comment by sswapnesh** [comment]
Hello @abdounasser202  

Initially we had fewer apps to publish here but now it seems we might have few more in future, which will make main README longer (with images too). @Yenthe666  IMO it's good idea to split it module wise.

File: README.md [hunk 1]
```diff
@@ -1,54 +1,8 @@
...
-You can view/edit/manage all aliases from Sales > Configuration > Product Aliases:
-![image](https://user-images.githubusercontent.com/6352350/177359471-fa9e6b8a-017e-4c1a-8629-d99c48d85973.png)
-They are only editable, creatable and deletable by sales administrators. Normal users can only read them.
-
+- [webshop_public_prices](webshop_public_prices/README.md): allow configuring to hide or show product prices and add to cart button for public users.
+- [website_featured_products](website_featured_products/README.md): allow to define which products should be featured when using the "Products" snippet.
+- [webshop_quick_sell_product_accessory](webshop_quick_sell_product_accessory/README.md): allow to quick add accessories to the basket from the product (in the webshop)
+- [product_alias](product_alias/README.md): allow to set aliases on products so they can be found with multiple names.
+- **webshop_order_customer_reference**: Customers can add their own reference on a webshop order
+- **webshop_quick_sell_product_accessory_public_prices**: Quick Sell Product accessories with public prices
```
Comment (Yenthe666) [inline README.md]: These two last apps shouldn't be in bold

**Review by Yenthe666 — CHANGES_REQUESTED** [review]
Just one note - rest looks good

File: README.md (hunk 1 above)
Comment (sswapnesh) [reply README.md]: `webshop_order_customer_reference` and `webshop_quick_sell_product_accessory_public_prices` doesn't have and README, maybe we should add with some text/images

Comment (Yenthe666) [reply README.md]: Would be ideal yes, but either way it does not need the ** for sure.

Comment (abdounasser202) [reply README.md]: Bold removed as suggested @Yenthe666

**Review by Yenthe666 — APPROVED** [review]

**Comment by Yenthe666** [comment]
Thank you 🙏

**Pull request merged**